src/App.js

```diff
@@ -6,25 +6,37 @@ import posts from './api/posts';
 import comments from './api/comments';
 import users from './api/users';
 
+import { PostList } from './components/PostList';
+
+const preparePosts = posts.map(post => ({
+  ...post,
+  user: users.find(user => user.id === post.userId),
+  usersComments: comments.filter(comment => comment.postId === post.id),
```

Comment:
давай упростим название
```suggestion
  comments: comments.filter(comment => comment.postId === post.id),
```